fdbclient/NativeAPI.actor.cpp

```diff
@@ -874,7 +876,11 @@ void setNetworkOption(FDBNetworkOptions::Option option, Optional<StringRef> valu
 				throw invalid_option_value();
 			}
 			break;
-		case FDBNetworkOptions::KNOB: {
+	    case FDBNetworkOptions::TRACE_FILE_IDENTIFIER:
+		    validateOptionValue(value, true);
+		    networkOptions.traceFileIdentifier = value.get().toString();
```

Comment:
The one thing I'm wondering is whether we need to restrict the characters present in this string or limit its length. I'm thinking the answer to both questions should be yes, and for the former maybe we would restrict to numeric and ascii characters? Maybe also underscore? At the very least, it may be necessary for our current parsing logic to restrict the presence of `.` in the name, though we could double-check that if we wanted to allow them.

Reply:
`.` should be find as it's already used anyways right? I agree with the length limit.

Reply:
The problem is if we are expecting the `.` to delimit certain fields. Adding more `.`s adds more fields, and may confuse any parsing logic we have.

Reply:
Oh I see. Forbidden it is.

Reply:
Done.